src/js/slider.js

```diff
@@ -136,6 +136,7 @@ Slider.prototype.render = function(colorStr) {
     html = html.replace(/{{slider}}/, tmpl.slider);
     html = html.replace(/{{huebar}}/, tmpl.huebar);
     html = html.replace(/{{cssPrefix}}/g, options.cssPrefix);
+    html = html.replace(/{{random}}/g, Math.random());
```

Comment:
Sorry for the late review :(

In my opinion, I think using an increasing integer value (for example, stored as the global variable) rather than `Math.random()`. Because `Math.random()` is too long and meaningless for ID.

Also, it would be better to assign a unique value to the **ColorPicker instance** not a slider. It can prevent other relevant issues.